INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
ERROR:webserver.webserver:Exception on /barcode_scanned [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/drinks-scanner-display/webserver/webserver.py", line 33, in barcode_scanned
    user = Users.get_active_user()
TypeError: get_active_user() takes exactly 1 argument (0 given)
INFO:werkzeug:127.0.0.1 - - [29/May/2016 22:20:34] "GET /barcode_scanned?barcode=123 HTTP/1.1" 500 -
INFO:werkzeug: * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
